City_CRUD_API/schemas.py

```diff
@@ -0,0 +1,38 @@
+from typing import Optional
+
+from pydantic import BaseModel
+
+
+class CityBase(BaseModel):
+    name: str
+    additional_info: str
+
+
+class CityCreate(CityBase):
+    pass
+
+
+class CityList(CityBase):
+    id: int
+
+    class Config:
+        # from_attributes = True
+        orm_mode = True
+
+
+class CityDelete(CityBase):
+    id: int
+
+    class Config:
+        # from_attributes = True
+        orm_mode = True
+
+
+class CityUpdate(CityBase):
+    id: int
```

Comment:
In the `CityUpdate` class, the `id` field is included, but typically updates are identified by path parameters rather than in the request body. Consider removing the `id` field from this schema if it's not used in the request body.